apps/accept_websocket/lib/accept/websocket.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Accept.Websocket do
+  use Definition, schema: Accept.Websocket.V1
+
+  @derive Jason.Encoder
+  defstruct version: 1,
+            path: nil,
+            port: nil,
+            idle_timeout: 180_000,
+            hibernate: false
+
+  defimpl Accept.Connection, for: __MODULE__ do
+    def connect(accept, opts) do
+      {
+        Accept.Websocket.Supervisor,
+        :start_link,
+        [
+          path: accept.path,
+          port: accept.port,
+          idle_timeout: accept.idle_timeout,
+          hibernate: accept.hibernate
+        ] ++ opts
```

Comment:
Is `++` what you want here? Or `Keyword.merge` with your specific configuration overriding anything that is set in `opts`?

Reply:
i could get behind that; they're not intended to overlap since the `opts` is where we pass in the stuff common to all receive gateways (the writer function, the batch size, and the timeout for sending messages even if a batch isn't full, not to be confused with the websocket idle_timeout for when the socket hasn't seen activity for a while.

Reply:
giving it a try; doesn't seem to make a difference so i'll allow it.